packages/query-core/package.json

```diff
@@ -5,37 +5,36 @@
 	"keywords": [],
 	"author": "Oscar Beaumont",
 	"license": "MIT",
-	"type": "module",
-	"source": "src/index.tsx",
-	"main": "dist/cjs/index.js",
-	"module": "dist/esm/index.js",
-	"types": "dist/types/index.d.ts",
-	"files": [
-		"dist"
-	],
+	"main": "dist/index.js",
 	"exports": {
+		"./package.json": "./package.json",
 		".": {
-			"import": "./dist/esm/index.js",
-			"browser": "./dist/esm/index.js",
-			"require": "./dist/cjs/index.js",
-			"node": "./dist/cjs/index.js"
+			"import": "./dist/index.mjs",
+			"default": "./dist/index.js"
 		}
 	},
+	"types": "dist/index.d.ts",
+	"files": ["dist"],
 	"scripts": {
-		"dev": "rollup -c -w",
-		"build": "rollup -c",
-		"prepare": "rollup -c"
+		"dev": "tsup --watch",
+		"build": "tsup",
+		"prepare": "tsup"
 	},
 	"dependencies": {
-		"@rspc/client": "workspace:*"
```

Comment:
Does this not need to exist? It should probs be a peer dep though?

Reply:
I'm not sure, solid-query has query-core as a direct dependency, which feels similar to this